joint_state_publisher/joint_state_publisher/joint_state_publisher.py

```diff
@@ -139,38 +139,116 @@ def init_urdf(self, robot):
                     joint['continuous'] = True
                 self.free_joints[name] = joint
 
-    def __init__(self):
-        description = get_param('robot_description')
-
-        self.free_joints = {}
-        self.joint_list = [] # for maintaining the original order of the joints
-        self.dependent_joints = get_param("dependent_joints", {})
-        self.use_mimic = get_param('use_mimic_tags', True)
-        self.use_small = get_param('use_smallest_joint_limits', True)
-
-        self.zeros = get_param("zeros")
-
-        self.pub_def_positions = get_param("publish_default_positions", True)
-        self.pub_def_vels = get_param("publish_default_velocities", False)
-        self.pub_def_efforts = get_param("publish_default_efforts", False)
-
+    def configure_robot(self, description):
+        self.node.get_logger().info('Got description, configuring robot')
```

Comment:
this feels more like `debug` than `info` to me

Reply:
Sure.  Fixed in 2917803.